src/utils/outputStream.ts

```diff
@@ -4,10 +4,10 @@ import path from "path";
 
 const ts = Date.now().toString();
 
-export const createSuccessStream = () =>
+export const createSuccessStream = (isVestingContract?: boolean) =>
```

Comment:
ContractType = "vesting" | "instant" that defaults to "instant" would be more readable here